.github/workflows/npmpublish.yml

```diff
@@ -7,24 +7,43 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+
     steps:
-      - uses: actions/checkout@v1
-      - uses: actions/setup-node@v1
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Setup Node
+        uses: actions/setup-node@v1
         with:
           node-version: 12
-      - run: npm ci
-      - run: npm run test
+
+      - name: Install
+        run: npm ci
+
+      - name: Test
+        run: npm run test
+
+      - name: Coverage
+        run: npm run coverage
```

Comment:
You have to make rely that section on the development build if coverage need sourceMap to be enabled.

Reply:
Going to remove sourceMap